fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -963,189 +970,249 @@ double MultiVersionDatabase::getMainThreadBusyness() {
 	return 0;
 }
 
-// Returns the protocol version reported by a quorum of coordinators
+// Returns the protocol version reported by the coordinator this client is connected to
 // If an expected version is given, the future won't return until the protocol version is different than expected
 ThreadFuture<ProtocolVersion> MultiVersionDatabase::getServerProtocol(Optional<ProtocolVersion> expectedVersion) {
```

Comment:
Can this future become ready if the server is older than 6.2? Is that documented anywhere? I think we're technically exposing this through the c api so maybe the documentation could go there.

Reply:
Yes, but not if it's older than 5.1. I can add a note about that.